htdocs/js/instrument_controlpanel_control.js

```diff
@@ -1,3 +1,21 @@
 $(function() {
     $('[data-toggle="tooltip"]').tooltip();
 });
+
+function swalFunction(event) {
```

Comment:
@pierre-p-s does the swalFunction need to be in both files:
`htdocs/js/instrument_controlpanel_control.js`
`htdocs/js/instrument_controlpanel_swaldeletedata.js `

Reply:
No I forgot to remove it

Reply:
removed